tests/v2/test_enr_utils.nim

```diff
@@ -39,12 +40,12 @@ procSuite "ENR utils":
       # Empty
       empty.toMultiAddresses().len() == 0
 
-  asyncTest "Init ENR for Waku Usage":
+  test "Init ENR for Waku Usage":
     # Tests RFC31 encoding "happy path"
     let
       enrIp = ValidIpAddress.init("127.0.0.1")
       enrTcpPort, enrUdpPort = Port(61101)
-      enrKey = wakuenr.crypto.PrivateKey.random(Secp256k1, rng[])[]
+      enrKey = generateSecp256k1Key()
```

Comment:
```suggestion
      enrKey = generateKey()
```
As per other usages of `crypto.PrivateKey.random(Secp256k1, rng[])[]`
no strong opinion here though, I like that it's explicit with the curve now